ads/index.js

```diff
@@ -87,6 +91,23 @@ function onAdsComplete (event) {
 	}
 }
 
+function sendAdLoadedTrackingEvent() {
+	const marks = performance.getEntriesByType ?
+		performance.getEntriesByName('firstAdLoaded')
+			.reduce((marks, mark) => {
+				marks[mark.name] = Math.round(mark.startTime);
+				return marks;
+			}, {}) :
+		{};
+
+	broadcast('oTracking.event', {
+		category: 'page-load',
```

Comment:
`category: 'ads', action: 'first-load'` is probably more obvious, as you already have `ads:blocked`